bindings/java/java/src/main/java/org/ethereum/evmc/EvmcLoaderException.java

```diff
@@ -0,0 +1,10 @@
+// EVMC: Ethereum Client-VM Connector API.
+// Copyright 2019-2020 The EVMC Authors.
+// Licensed under the Apache License, Version 2.0.
+package org.ethereum.evmc;
+
+public class EvmcLoaderException extends Exception {
```

Comment:
Consider making it extend RuntimeException - this way it's an unchecked exception.

Reply:
But perhaps I don't fully understand the distinction between `Exception` and `RuntimeException` -- could you explain please?

Reply:
Sure thing. In Java, you _must_ catch checked exceptions, or the code doesn't compile. You can let unchecked exceptions "unchecked", meaning you don't need to catch them explicitly. They still happen and bubble up.
The net difference is that sometimes your Java code ends up looking like a bunch of try/catch statements or throws.
This is a subject of hot debate and frankly, I'm fine either way. I just prefer `RuntimeException`s myself.
https://stackoverflow.com/questions/6115896/understanding-checked-vs-unchecked-exceptions-in-java

Reply:
Thanks! My argument for checked exception here is that this EVMC-specific loading error should be handled at the place this is integrated in, likely it would be translated into some local exception in the project?

Though I don't have a very strong opinion, fine either way.

Reply:
OK, let’s do it.